guides/v2.1/coding-standards/code-standard-demarcation.md

```diff
@@ -471,7 +471,9 @@ jQuery('#{$htmlId}-suggest').treeSuggest({$selectorOptions});
 <li>Reduces the number of files to be modified.</li></ul>
 
 **Acceptable PHP file**
-<pre>
+
+{% highlight php startinline=true %}
```

Comment:
For code highlighting where there isn't a preceding `php` in the code snippet (like in this instance) use the following format for code:

```php?start_inline=1
<php code without starting <php? tag>
```

Reply:
@shrielenee need to write _?start_inline=1_ text?

Reply:
![screen shot 2018-06-07 at 11 15 03 am](https://user-images.githubusercontent.com/14031342/41112243-0ba35176-6a44-11e8-8c6c-902c7b26e4ac.png)

Reply:
I had to reply with a screenshot because its difficult to escape formatting here!